Formula/shogimaru.rb

```diff
@@ -0,0 +1,30 @@
+class Shogimaru < Formula
+  desc "Shogi GUI supporting USI protocol"
+  homepage "https://shogimaru.com/index.en.html"
+  url "https://github.com/shogimaru/shogimaru/archive/refs/tags/v1.3.0.tar.gz"
+  sha256 "152f18e381e6d08abae8a494002387a9ac2786104c8c5e98ca2c4fd011e11bca"
+  license "MIT"
+  head "https://github.com/shogimaru/shogimaru.git", branch: "develop"
+
+  depends_on "qt"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    system "qmake", "-recursive", "CONFIG+=release", "target.path=#{bin}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    if OS.mac?
+      otool "-L", "#{bin}/shogimaru"
```

Comment:
I don't think this will really validate that it works correctly though.

Reply:
I have made first Formula pull-request on Linux, I don't have a mac.

Could you tell me how to validate a GUI application on brew?
Should it be checked if it is possible to start the application?

Reply:
That would be a better test, yes.